projects/ad9213_evb/common/ad9213_evb_bd.tcl

```diff
@@ -0,0 +1,178 @@
+
```

Comment:
Please remove the first and the last line which are empty.
Also, could you add the copyright and license header?